src/runtime_src/core/pcie/driver/linux/xocl/subdev/ert_user.c

```diff
@@ -19,6 +19,7 @@
 #include "kds_client.h"
 
 #define	ERT_MAX_SLOTS		128
+#define	ERT_MIN_SLOT_SIZE	512
```

Comment:
I would suggest do not define this macro.
The minimum slot size is defined by "ert_user->cq_range / ERT_MAX_SLOTS". If there is a shell has different cq_range, use this formula would still work.